src/github.com/matrix-org/rageshake/submit_test.go

```diff
@@ -215,6 +227,23 @@ func checkParsedMultipartUpload(t *testing.T, p *payload) {
 	}
 }
 
+func stringSlicesEqual(got, want []string) bool {
+	if (got == nil) != (want == nil) {
```

Comment:
`nil` slices have a length of 0, so unless you really really care about the distinction between a `nil` slice and an empty slice, I would skip this check and rely on `len()` instead. `nil` slices are also `range`able so in the case where they are both 0 then the `for` loop body won't execute. If we do care about the distinction e.g there is a subtle semantic difference in your code between the two, I'd rather we modified the code such that it didn't matter.